adapters/repos/db/vector/hnsw/delete.go

```diff
@@ -585,6 +586,15 @@ func (h *hnsw) removeTombstonesAndNodes(deleteList helpers.AllowList, breakClean
 			h.Unlock()
 			if h.compressed.Load() {
 				h.compressedVectorsCache.delete(context.TODO(), id)
+
+				Id := make([]byte, 8)
```

Comment:
nitpicking (feel free to ignore): In go it's not common to use capitalized variable names, especially if it clashes if another variable with the same name but lowercase. Something like `idBuf` would be more idiomatic